frontend/src/components/GoalCards/GoalCard.js

```diff
@@ -0,0 +1,205 @@
+import ObjectiveCard from './ObjectiveCard';
+import ObjectiveButton from './components/ObjectiveButton';
+import Topics from './components/Topics';
+import './GoalCard.scss';
+import colors from '../../colors';
+
+function GoalCard({
+  goal,
+  recipientId,
+  regionId,
+  showCloseSuspendGoalModal,
+  performGoalStatusUpdate,
+}) {
+  const {
+    id, // for keys and such, from the api
+    ids, // all rolled up ids
+    goalStatus,
+    createdOn,
+    goalText,
+    goalTopics,
+    objectiveCount,
+    reasons,
+    objectives,
+    previousStatus,
+  } = goal;
+
+  const endDates = useMemo(() => objectives.reduce((prev, curr) => [
+    ...prev,
+    ...curr.activityReports.map((ar) => ar.endDate),
+  ], []).sort((a, b) => moment(a, 'MM/DD/YYYY') - moment(b, 'MM/DD/YYYY')), [objectives]);
+
+  const lastTTA = endDates.pop();
+
+  const history = useHistory();
+
+  const goalNumbers = goal.goalNumbers.join(', ');
+
+  const onUpdateGoalStatus = (newStatus) => {
+    if (newStatus === 'Completed' || newStatus === 'Closed' || newStatus === 'Ceased/Suspended' || newStatus === 'Suspended') {
+      // Must provide reason for Close or Suspend.
+      showCloseSuspendGoalModal(newStatus, ids, goalStatus);
+    } else {
+      performGoalStatusUpdate(ids, newStatus, goalStatus);
+    }
+  };
+
+  const expandObjectivesRef = useRef();
+
+  const [objectivesExpanded, setObjectivesExpanded] = useState(false);
+
+  const determineFlagStatus = () => {
+    const reasonsToWatch = reasons.find((t) => reasonsToMonitor.includes(t));
+    if (reasonsToWatch) {
+      return (
+        <>
+          <Tooltip
+            displayText={<FontAwesomeIcon className="margin-left-1" size="1x" color={colors.error} icon={faFlag} />}
+            screenReadDisplayText={false}
+            buttonLabel={`Reason for flag on goal ${goalNumbers} is monitoring. Click button to visually reveal this information.`}
+            tooltipText="Related to monitoring"
+            hideUnderline
+          />
+        </>
+      );
+    }
+    return null;
+  };
+
+  const closeOrOpenObjectives = (collapseFromObjectives) => {
+    if (collapseFromObjectives && expandObjectivesRef.current) {
+      expandObjectivesRef.current.focus();
+    }
+
+    setObjectivesExpanded(!objectivesExpanded);
+  };
+
+  const contextMenuLabel = `Actions for goal ${id}`;
+  const showContextMenu = true;
+  const menuItems = [
+    {
+      label: 'Edit',
+      onClick: () => {
+        history.push(`/recipient-tta-records/${recipientId}/region/${regionId}/goals/${id}`);
+      },
+    },
+  ];
+
+  return (
+    <article className="ttahub-goal-card usa-card margin-x-3 margin-y-2 padding-3 radius-lg border smart-hub-border-base-lighter ">
+      <div className="display-flex flex-justify">
+        <StatusDropdown
+          goalId={id}
+          status={goalStatus}
+          onUpdateGoalStatus={onUpdateGoalStatus}
+          previousStatus={previousStatus}
+          regionId={regionId}
+        />
```

Comment:
I noticed that when I don't have permissions to edit or create goals, the status text in the `StatusDropdown` ends up getting flexed to the center:

![Screen Shot 2022-09-08 at 9 55 56 AM](https://user-images.githubusercontent.com/17074357/189169026-cfeefa3d-e67d-40cc-b4ad-62a6814e85fb.png)

Reply:
Thanks for noticing that! I believe [that issue has been addressed](https://github.com/HHS/Head-Start-TTADP/pull/1027/commits/fcfddbfe99debe4d6ea7a29cd6c7407ef1c911ae).